apps/server/priv/boot.exs

```diff
@@ -0,0 +1,17 @@
+[script_dir] = System.argv()
+
+[script_dir, "..", "lib", "*.ez"]
+|> Path.join()
+|> Path.wildcard()
+|> Enum.each(fn archive_path ->
+  lib =
+    archive_path
+    |> Path.basename()
+    |> String.replace_suffix(".ez", "")
```

Comment:
does this work? I think you need the whole file with the `.ez` extension and everything.

Reply:
As discussed on Discord, the `.ez` part is still contained in `archive_path`, so it gets included in the ebin path starting on line 12